vue-frontend/src/components/contact/ThankyouScheduleMeeting.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <section class="tw-relative">
+    <img
+      :src="ctaimage400w"
```

Comment:
Images should be in the container after 2k or 2.5 devices, the same as other sections.
its looking weird currently